app/modules/encounters/parameters.py

```diff
@@ -97,7 +97,16 @@ def replace(cls, obj, field, value, state):
             obj.comments = value
             ret_val = True
         elif field == 'customFields':
-            obj.custom_fields = value
+            # taken directly from edm code:
```

Comment:
Are we sure that the FE actually uses this? It does seem quite complex but if it's needed, good catch.

Reply:
asked in slack.  response from ben reads to me as "keep it":

> not used now, but actually it's what i would prefer to use in the future
>   _[[slack](https://wildme.slack.com/archives/C035JH4G5D4/p1659126186662049?thread_ts=1659123773.064699&cid=C035JH4G5D4)]_